doc/Developers-Intro.md

```diff
@@ -61,6 +61,8 @@ Here's a few primers if you are new to Friendica or to the PSR-2 coding standard
 Don't worry, you don't have to know by heart the PSR-2 coding standards to start contributing to Friendica.
 There are a few tools you can use to check or fix your files before you commit.
 
+For documentation we use the standard of *one sentence per line* for the `md` files in the `/doc` and `/doc/$lng` subdirectories.
```

Comment:
See coding standards for the docs. One line per sentence, finding long lines is one of the goals for this rule, so one can find shorter sentences for the long line. The markdown files are rendered by Friendica and github on display anyway.